plugins/nginx-vhosts/src/pagesize/pagesize.go

```diff
@@ -0,0 +1,10 @@
+package main
+
+import (
+	"fmt"
+	"os"
+)
+
+func main() {
+	fmt.Println(os.Getpagesize())
```

Comment:
Will this value break tests if we give the server more resources? Just curious, haven't run it locally yet.